exercises/pangram/stack.yaml

```diff
@@ -0,0 +1 @@
+resolver: lts-8.6
```

Comment:
Ah, I just read the [instructions](https://github.com/exercism/xhaskell#exercise-structure) again and realised I may have misunderstood this bit:

>* `stack.yaml` has just one line specifying the current [Stack snapshot](https://www.stackage.org/snapshots). We use the same resolver for all the exercises.

The first part suggests we should include the current Snapshot from the link given, but the second part says we should use the same resolver for all exercises!!!11one

I'd prefer consistency within the track, so either downgrade this one to `lts-8.2`, or upgrade the others.

Perhaps we could fix the resolver version in a config file within `xhaskell` and then create a bin  in `xhaskell/bin` that will generate the `stack.yaml` for a given path? Starting with `stack.yaml`, this could evolve to a file that would generate the full exercise structure.